Source/Scene/Billboard.js

```diff
@@ -1149,6 +1172,49 @@ define([
     };
 
     /**
+     * Gets a billboard's screen space bounding box centered around screenSpacePosition.
+     * @param {Billboard} billboard The billboard to get the screen space bounding box for.
+     * @param {Cartesian2} screenSpacePosition The screen space center of the label.
+     * @param {BoundingRectangle} [result] The object onto which to store the result.
+     * @returns {BoundingRectangle} The screen space bounding box.
+     *
+     * @private
+     */
+    Billboard.getScreenSpaceBoundingBox = function(billboard, screenSpacePosition, result) {
```

Comment:
Can you add unit tests for all the new `getScreenSpaceBoundingBox`?

Also, although I am all for keeping the API surface area small, have folks asked for these functions before?  It could be worth adding public `prototype` functions for the public API?